.eslintignore

```diff
@@ -0,0 +1 @@
+__mocks__
```

Comment:
Not a suggestion, but a question: why?

Reply:
Mocks normally have to abuse the guidelines quite heavily, to mock a class. We could have some mock specific rules, though. Just not that restrictive.

Reply:
Got it, thanks!